rules-tests/Php80/Rector/Class_/AnnotationToAttributeRector/config/configured_rule.php

```diff
@@ -22,6 +22,8 @@
                 new AnnotationToAttribute('Doctrine\ORM\Mapping\Table'),
                 new AnnotationToAttribute('Doctrine\ORM\Mapping\Id'),
                 new AnnotationToAttribute('Doctrine\ORM\Mapping\Column'),
+                new AnnotationToAttribute('Doctrine\ORM\Mapping\OrderBy'),
+                new AnnotationToAttribute('Doctrine\ORM\Mapping\OneToMany'),
                 new AnnotationToAttribute('ApiPlatform\Core\Annotation\ApiResource'),
```

Comment:
It would be better to use already existing classes, as the class name is not related to syntax changes. E.g. Table

The less inputs we have, the easier to find potential bugs

Reply:
I can't found example of using `Table` yet for usage of missplaced comma with next line `})`.

```php
",    
})
```

This test is for that example.

Reply:
It's this file: https://github.com/rectorphp/rector-src/commit/3157491c7808b8ca110ada5a9a5cd64206972221#diff-2ee5291f5328e43dbbee108bb9e267c29de6527ed96af06b5305044195d361ab

The test is not a problem. But a new class for each new test is.
Ideally there should be just one configuration here:

```php
new AnnotationToAttribute('Rector\...Tests\CustomAnnotation'),
```

Reply:
Ok, do you think it needs to be reverted?

Reply:
I reverted at https://github.com/rectorphp/rector-src/pull/221